packages/lit-dev-content/site/docs/v3/releases/upgrade.md

```diff
@@ -1,40 +1,69 @@
 
-- Drops support for IE11
-- Is published as ES2021
-- Removes a few deprecated APIs
+- IE11 is no longer supported.
+- Lit's npm modules are now published as ES2021.
+- APIs marked deprecated in Lit 2.x releases have been removed.
+- SSR hydration support modules have moved to the `@lit-labs/ssr-client` package.
+- Decorator behavior has been unified between TypeScript experimental decorators and standard decorators.
+- Support was removed for Babel decorators version "2018-09"
 
-Lit 3.0 should require no changes to upgrade from Lit 2.0 for the vast majority
+Lit 3.0 should require no changes to upgrade from Lit 2.x for the vast majority
 of users. Most apps and libraries should be able to extend their npm version
 ranges to include both 2.x and 3.x, like `"^2.7.0 || ^3.0.0"`.
 
-Lit 2.x and 3.0 are _interoperable_: templates, base classes, directives,
-decorators, etc., from one version of Lit will work with those from another.
+Lit 2.x and 3.0 are _interoperable_: templates, base classes, and directives from one version of Lit will work with those from another.
+
+## Updates to Lit decorators
+
+The Lit 3 decorators are largely backwards compatible. If you've been using them with TypeScript, **most likely no changes are needed**. We've added forward compatibility with standard decorators, but the legacy `experimentalDecorators: true` without the `accessor` keyword is currently the more performant option.
+
+In Lit 3.0 there are three supported ways to use our decorators:
+
+1. With the TypeScript `experimentalDecorators` flag, and no `accessor` keyword. This is backwards compatible with Lit 2.
+2. With the TypeScript `experimentalDecorators` flag and using the `accessor` keyword. This aids migration to standard decorators. [Code can add the `accessor` keyword one decorator at a time to migrate gradually](#standard-decorator-migration).
+3. As standard decorators, which need the `accessor` keyword, and TypeScript 5.2 (without the `experimentalDecorators` flag), or Babel 7.23.0.
+
+Each supported way of using decorators is self contained. An application can mix and match libraries written in each of these configurations, and it is not generally a breaking change for a library to switch from one decorator configuration to another.
+
+To make Lit decorators consistent between both decorator modes there have been some minor breaking
+changes to Lit decorator behavior:
+
+- `@property()` and `@state()` must be used on the setter for hand-written accessors. [Decorating the getter no longer works](#decorated-getter).
+- `requestUpdate()` is called automatically for `@property()` and `@state()` decorated
+  accessors where previously that was the setters responsibility.
+- The value of an accessor is read on first render and used as the initial value for `changedProperties` and attribute reflection.
+- Lit 3 decorators do not support Babel (`@babel/plugin-proposal-decorators`) with `version: "2018-09"`.
```

Comment:
I think it would be good to also state the alternative.
```suggestion
- Lit 3 decorators no longer support `version: "2018-09"` option of `@babel/plugin-proposal-decorators`. Babel users should instead [migrate](#standard-decorator-migration) to use `version: "2023-05"` with plugin version greater than 7.23.0 which follows the TC39 standard decorator spec.
```

Reply:
We should also say that this requires using the `accessor` keyword too